tools/helmify/src/config.yaml

```diff
@@ -72,10 +72,10 @@ oidc-authservice:
 
 cluster-local-gateway:
   kustomization_paths:
-    - upstream/common/istio-1-14/cluster-local-gateway/base
+    - upstream/common/istio-1-16/cluster-local-gateway/base
   output_helm_chart_path: charts/common/cluster-local-gateway
-  version: 0.1.0
-  app_version: 1.6.1
+  version: 0.1.1
```

Comment:
Why is this a patch version increase, when istio is a minor version increase?

Reply:
good catch, addressed.